Domain/Sources/Domain/extension/UserCodableExtension.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  File.swift
+//  
+//
+//  Created by Hassan on 25/03/2023.
+//
+
+import Foundation
+import Core
+extension UserCodable {
```

Comment:
plz don't use extension UserCodable you can create new struct UserMapper and create func to map user from codable to User